.github/workflows/integration-test.yml

```diff
@@ -1170,3 +1170,23 @@ jobs:
           timeout_minutes: 8
           retry_wait_seconds: 5
           command: cd terraform/stress && terraform destroy --auto-approve
+          
+  E2EEKSTest:
```

Comment:
nit: Technically the format is PLATFORM<TestCase>Test. Could do `EKSEndToEndTest` for example